presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestOraclePoolDistributedQueries.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.oracle;
+
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.tpch.TpchTable;
+
+public class TestOraclePoolDistributedQueries
```

Comment:
IMO running whole `DistributedQueries` is superfluous. It adds a lot of test runtime. `TestOraclePoolIntegrationSmokeTest` should be enough.

Reply:
done